tests/lib/authentication.py

```diff
@@ -48,3 +47,42 @@ def generate_int_authentication():
             int_token = (f"Bearer {details.get('access_token')}")
 
         return int_token
+
+    @staticmethod
+    def generate_prod_authentication():

```

Comment:
I think you can dedupe this static method with the previous `generate_int_authentication` method so we are a little more DRY here.